listenbrainz/tests/integration/test_api.py

```diff
@@ -66,6 +66,11 @@ def test_get_listens(self):
         # check for latest listen timestamp
         self.assertEqual(data['latest_listen_ts'], ts)
 
+        # request with min_ts should work
+        response = self.client.get(url, query_string = {'min_ts': int(time.time())})
+        self.assert200(response)
+
```

Comment:
It'd also be good to have a test where you select a max_ts lower than the most recent listen to ensure that you get both the listens that you query for and the timestamp of the most recent one.